client/src/pages/CreateUserProfile.js

```diff
@@ -169,8 +170,38 @@ const CreateProfile = ({ email, firstName, lastName, history }) => {
     createUserFormReducer,
     initialState,
   );
+  useEffect(() => {
+    (async function linkAccounts() {
+      try {
+        const resAccounts = await axios.get("/api/auth/link-accounts");
```

Comment:
Is there a reason for why getting the Auth0 user IDs and linking the Auth0 user IDs are separate API endpoints? Could we not combine them into one call?

Reply:
Yes Manny, we were thinking to let user decide link or not at the beginning then decided not to do that  -> [link](https://fightpandemics.slack.com/archives/C014E5X0FNY/p1605111386057900?thread_ts=1604939904.055700&cid=C014E5X0FNY) <- these 2 endpoints can be combined together, I will combine them as one.